Container/CSS/card.css

```diff
@@ -0,0 +1,143 @@
+@import url(/Container/CSS/style.css);
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body{
+    display: flex;
+    gap: 1rem;
+    flex-wrap: wrap;
+}
+
+.card{
+    display: flex;
+    flex-direction: column;
+    background: #191938;
+    gap: 0.3rem;
+    box-shadow: 0 2px 3px  4px var(--alt-text-color);
+    margin: 1rem;
+    width: 17.75rem;
+    outline: 1px solid var(--alt-text-color);
+    border-radius: 1rem;
+}
+.horizontal{
+    display: flex;
+    flex-direction: row;
+    background: #191938;
+    gap: 0.3rem;
+    box-shadow: 0 2px 3px  4px var(--alt-text-color);
+    margin: 1rem;
+    width: 30.75rem;
+    height: 12rem;
+    outline: 1px solid var(--alt-text-color);
+    border-radius: 1rem;
+}
+.badge-card{
+    height: fit-content;
+}
+.card-images{
+    width: 100%;
+    height: 13.25rem;
+    border-radius: 1rem 1rem 0rem 0rem;
+    padding-bottom: 1.4rem;
+}
+.text-context{
+    font-family: var(--font-primary);
+    padding: 0.1rem 0.8rem;
+    color: var(--text-color);
+}
+.details-heading{
+    font-size: 1.6rem;
+    font-weight: 700;
+}
+
+.details-price{
+    font-size: 1rem;
+    font-weight: 500;
+}
+.details-content{
+    border-top: 2px solid var(--text-color);
+    font-size: 0.875rem;
+    margin: 0.3rem 0.8rem;
+    padding: 0.8rem 0rem;
+    color: var(--text-color);
+    border-bottom: 2px solid var(--text-color);
+}
+.details-contents{
```

Comment:
Here you can create a class that has the same properties with details-content and reuse it.
and just need to define other 

classes that have different properties from details-content.

Reply:
my class names are already too huge in the div, ill see what can be done